### PR TITLE
[KOGITO-8115] added Nexus repository for nightly deploy job

### PR DESCRIPTION
**Pull request by mbiarnes**
https://github.com/kiegroup/kogito-pipelines/pull/696
https://github.com/kiegroup/kogito-runtimes/pull/2616

### CODE REVIEW

File: dsl/config/branch.yaml [hunk 1]
```diff
@@ -60,6 +60,9 @@ maven:
     staging_profile_id: 1976dd1c18392a
     build_promotion_profile_id: ea49ccd6f174
   artifacts_repository: ''
+  local_zipped_artifacts_repository:
+    url: https://repository.jboss.org/nexus/content/repositories/snapshots/
+    creds_id: kie-ci1-upload
```
Comment (radtriste) [inline dsl/config/branch.yaml]: I think you should use the already existing `artifacts_repository` and add a `artifacts_repository_credentials_id`
```suggestion
  artifacts_repository: https://repository.jboss.org/nexus/content/repositories/snapshots/
  artifacts_repository_credentials_id: kie-ci1-upload
```

**Review by radtriste — CHANGES_REQUESTED** [review]

File: dsl/config/branch.yaml (hunk 1 above)
Comment (radtriste) [reply dsl/config/branch.yaml]: forget about this

File: dsl/config/branch.yaml [hunk 2]
```diff
@@ -60,6 +60,9 @@ maven:
     staging_profile_id: 1976dd1c18392a
     build_promotion_profile_id: ea49ccd6f174
   artifacts_repository: ''
+  local_zipped_artifacts_repository:
```
Comment (radtriste) [inline dsl/config/branch.yaml]:
```suggestion
  artifacts_upload_repository:
```

**Comment by mbiarnes** [comment]
@radtriste comments resolved

**Review by radtriste — APPROVED** [review]

**Pull request merged**